atomicfu-transformer/src/main/kotlin/kotlinx/atomicfu/transformer/AtomicFUTransformerJS.kt

```diff
@@ -15,12 +15,12 @@ private const val ATOMIC_CONSTRUCTOR = """(atomic\$(ref|int|long|boolean)\$|Atom
 private const val ATOMIC_ARRAY_CONSTRUCTOR = """Atomic(Ref|Int|Long|Boolean)Array\$(ref|int|long|boolean|ofNulls)"""
 private const val MANGLED_VALUE_PROP = "kotlinx\$atomicfu\$value"
 
-private const val TRACE_CONSTRUCTOR = "atomicfu\\\$Trace\\\$"
-private const val TRACE_APPEND = "atomicfu\\\$Trace\\\$append\\\$"
-private const val TRACE_NAMED = "atomicfu\\\$Trace\\\$named\\\$"
+private const val TRACE_CONSTRUCTOR = "Trace\\\$atomicfu\\\$"
+private const val TRACE_APPEND = """(Trace)\$(append)\$(1|2|3|4)\$(atomicfu)\$"""
```

Comment:
IDEA highlights this regexp and suggests replacing `(1|2|3|4)` with `[1234]`. Also, worth to drop a comment that this is a number of overloads